src/libraries/System.Data.Common/tests/System/Data/Common/DbCommandTests.cs

```diff
@@ -154,7 +154,6 @@ protected override DbDataReader ExecuteDbDataReader(CommandBehavior behavior)
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono, "GC has different behavior on Mono")]
```

Comment:
These should be changed to `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsPreciseGcSupported))]`. Otherwise, the test will fail intermittently with Mono conservative GC stack scanning.

Reply:
Oh nice, I wasn't aware we already have a PlatformDetection for this 👍 @MaximLipnin please change to use that instead.

Reply:
I'll take that into account, thanks!